client/state/selectors/get-current-plan-term.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Internal dependencies
+ */
+
+import { getPlan } from 'lib/plans';
+import { getSitePlan } from 'state/sites/selectors';
+
+/**
+ * Returns term of the active plan for given siteId, e.g. TERM_MONTHLY
+ *
+ * @param {Object} state Current state
+ * @param {Number} siteId Site ID
+ * @return {string} Human-readable interval type
+ */
+export default function getTermFromCurrentPlan( state, siteId ) {
+	const currentPlanProduct = getSitePlan( state, siteId );
+	if ( ! currentPlanProduct ) {
+		return null;
+	}
+
+	const currentPlanObject = getPlan( currentPlanProduct.product_slug );
+	if ( ! currentPlanObject ) {
+		return null;
+	}
+	return currentPlanObject.term;
```

Comment:
Any chance the current plan object might be missing a `term` property?

Reply:
No :-) every plan object defined in `lib/plans/constants` is expected to have it and there are separate tests for that.